### DISCUSSION TITLE
Focus.Achievement : Automation order

### DISCUSSION THREAD

**Discussion by Antoninqlc** [Farigh/pokeclicker-automation#354] [Q&A] [answered]
I finished all the Galar achievement and have the option for skipping magikarp jump quests but the automation try to finish it while I didn't got any Orre achievement, is that normal ? (sorry for my bad english)

**Comment by Farigh** [comment]
Hi,

The automation considers the ingame categories to sort achievements : 
![image](https://github.com/Farigh/pokeclicker-automation/assets/11090416/7fe9f82b-34a2-4d7c-9587-61133ebc0ed7)
For convenience, the `Sevii` category is considered as `Hoenn` difficulty, and the `Magikarp Jump` as `Galar` difficulty.

The option for the magikarp jump region is to complete the achievements of this sub-region last (not to skip it entirely).

If enabled, it will be perform last, otherwise, it will be performed at the same time as the Galar region ones.

If the automation is not performing Orre challenges (which should be in the Hoenn category) before the Magikarp Jump ones, you might not have acces to the corresponding area.

If you do, can you upload your save here, so I can take a closer look.

**Comment by Farigh** [comment, marked as answer]
The Orre prioritization should have been addressed by the #356 PR
